README.rst

```diff
@@ -106,6 +106,12 @@ the ``dict`` alternate construction. If you need to disable a default in
                 ref: refs/pull/14859/head
         target: acsone 9.0
 
+It's also possible to set default values to all repositories
+
+.. code-block:: bash
+
+    $ gitaggregate -c repos.yaml --with-default depth:1
```

Comment:
I'm wondering if such defaults would not be best located in the yaml file?

Reply:
Yeah, I thought so too.
But how? A special `defaults` key?

```yaml
defaults:
    depth: 1

product_attribute:
    remotes:
        oca: https://github.com/OCA/product-attribute.git
        acsone: git+ssh://git@github.com/acsone/product-attribute.git
    merges:
        - oca 8.0
        - oca refs/pull/105/head
        - oca refs/pull/106/head
    target: acsone aggregated_branch_name
```